### ISSUE TITLE
Requirements for sp110e not found

### ISSUE THREAD

**Issue by th3d00d** [roslovets/SP110E-HASS#4]
After installing this using HACS and adding the necessary content to the configuration.yaml it shows the following error when checking the configuration:

Platform error light.sp110e - Requirements for sp110e not found: ['sp110e==1.4.0'].

How can I avoid this error?

**Comment by danielbrunt57** [comment]
I have the same error...
```
Platform error light.sp110e - Requirements for sp110e not found: ['sp110e==1.4.0'].
```
Does this mean it is not seeing the SP110E controller??
I'm using Home Assistant Blue / Hardkernel Odroid-N2 running Home Assistant OS.
I've tried a Belkin F8T016 BT211 adapter, a CSR 4.0 Dongle and a FEASYCOM FSC-BP119.

**Comment by roslovets** [comment]
Fixed in [v1.0.1](https://github.com/roslovets/SP110E-HASS/releases/tag/v1.0.1). Tested in HASS 2022.8.6, HACS 1.26.2. Thanks

**Comment by danielbrunt57** [comment]
Mine is working now. Thank you!
